plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/database/DatabaseTransferUtils.java

```diff
@@ -335,13 +374,48 @@ private static DBEPersistAction[] generateStructTableDDL(DBRProgressMonitor moni
 
             containerMapping.setTargetName(tableFinalName);
 
-            List<DBEPersistAction> actions = DBExecUtils.getActionsListFromCommandContext(monitor, commandContext, executionContext, options, null);
-            return actions.toArray(new DBEPersistAction[0]);
+            actions.addAll(
+                DBExecUtils.getActionsListFromCommandContext(
+                    monitor,
+                    commandContext,
+                    executionContext,
+                    options,
+                    null));
+            return table;
         } catch (DBException e) {
             throw new DBException("Can't create or modify target table", e);
         }
     }
 
+    public static void implementPropertyChanges(
+        DBRProgressMonitor monitor,
+        Map<DBPPropertyDescriptor, Object> changedProperties,
+        DBECommandContext commandContext,
+        DatabaseMappingContainer containerMapping,
+        DBSEntity table) {
```

Comment:
The name of this method is kinda akward. Perhaps it should be called `applyPropertyChanges`?

Also, please specify all required annotations.

```suggestion
        DBSEntity table
    ) {
```